README.md

```diff
@@ -165,15 +165,15 @@ git checkout master
 8. Add the next minor release to the ``CHANGES`` file:
 
 ```
-## 1.10.0 (unreleased)
+## 1.11.0 (unreleased)
 ```
 
 9. Update the version in ``Client.php``:
 
 ```php
 class Raven_Client
```

Comment:
This class does not exists anymore in `2.x` and should be updated to reflect PSR-4 changes